Openmail/system1-cmp
# Pfe 4216

## Background
- [x] Adds new slim banner with feature flag
- [x] Adds full width mode with feature flag
- [x] Adds drop shadow toggle feature flag
- [x] Adds close with &times; buttom feature flag
- [x] Adds default background color override 

## Test Plan
```
yarn dev
```
Go to http://localhost:8080/tcf-2.0.html

1. Layer0 - Check "Use Slim"
- [x] Slim banner (layer 0) should load.  
- [x] "Learn More" Should navigate to Layer1.
- [x] "Ok, Got it" should accept all
- [x] Test layout on mobile

2. Full Width Mode - Check "Full Width"
- [x] Layer1 and Vendor list should span full width 
- [x] No rounded corners on Layer1 and Vendor list

3. Toggle Drop Shadow - (Un)Check "Drop Shadow"
- [x] When checked, all layers have drop shadow
- [x] When not checked, no layers have drop shadow

4. Close (&times;) button - Check "Close with &times;"
- [x] When checked, all banner layers should have &times; button displayed in upper right
- [x] When unchecked, not button should render
- [x] Clicking should "accept all" and close CMP UI
- [x] Check positioning on mobile UI

5. Background color override - Set config option theme: { backgroundColor: "#f1f1f1" }
- [x] Banner background colors should display configured color
- [x] No overlapping or nested UI elements should  overlay with another background color
